packages/editor/src/components/block-list/style.scss

```diff
@@ -860,6 +860,19 @@
 				top: $block-toolbar-height + $block-padding;
 			}
 
+			// Rules inside this query are only run by Microsoft Edge.
+			// Edge has bugs around position: sticky; combined with transform, so create different rules.
+			@supports (-ms-ime-align:auto) {
+				// Move the block toolbar upwards using margin, as translate doesn't work.
+				margin-top: -$block-toolbar-height -$block-padding -$border-width;
+
+				// Disable the technique used to move the block toolbar upwards in non-edge browsers.
+				transform: translateY(0);
```

Comment:
That is very puzzling indeed. Perhaps you are on a newer version of Edge than my VM, and perhaps that version of Edge fixes this. I'm working from my home office today so it's a bit harder for me to test, so I would love additional eyes on how this toolbar works across versions of Edge, because right now I can't reproduce.

Reply:
That is odd. I'm also now seeing that it looks fine in master.

Reply:
Hrm. That's interesting indeed. I'm adding "In progress" label for now, and removing milestone, just so we don't end up fixing something that's fixed in the browser itself. I will be sure to check the browser version in my coworking space tomorrow, see if it's an older one.

Reply:
Iiiiiinteresting, the plot thickens. Behold my Edge version:

<img width="282" alt="screen shot 2018-09-25 at 10 07 58" src="https://user-images.githubusercontent.com/1204802/46001286-353e0e00-c0ab-11e8-81a0-9d4a36ad1dec.png">

And still seeing the issue:

<img width="1049" alt="screen shot 2018-09-25 at 10 07 43" src="https://user-images.githubusercontent.com/1204802/46001295-3a9b5880-c0ab-11e8-80da-abb2cdd0c1ec.png">

Stay tuned as I download a new vbox image with a newer version and try again.

Reply:
Yay, it's fixed in the newer version:

<img width="872" alt="screen shot 2018-09-25 at 12 02 32" src="https://user-images.githubusercontent.com/1204802/46007685-f748e600-c0ba-11e8-8b48-a8ded6b2063c.png">

<img width="294" alt="screen shot 2018-09-25 at 12 02 41" src="https://user-images.githubusercontent.com/1204802/46007688-f9ab4000-c0ba-11e8-9527-37916aa725aa.png">

Since Edge auto updates, I'm ging to close this one as now obsolete. Wohoo! Thanks for the sanity checks.